### ISSUE TITLE
add tab "configure ports" to configuration dialog

### ISSUE THREAD

**Issue by jakoch** [WPN-XM/WPN-XM#327]
**Idea**
![laragon-services-and-ports](https://cloud.githubusercontent.com/assets/85608/6315979/d0fe2692-ba13-11e4-8256-ff15a0570522.png)

**Todo**
- [x] add tab "configure ports" to configuration dialog
    - [x] Nginx (via the Pools/Upstream/Server configuration)
    - [x] PHP (see above)
    - [x] xdebug
    - [x] MariaDb
    - [x] MongoDb
    - [x] PostgreSQL
    - [x] Redis
    - [x] Memcached
- [x] form field values alter the wpn-xm.ini file
- [x] set these changes to the individual daemon config files, too
  - The port values need to be written to nginx.conf for example and wpn-xm.ini 
  - ~~keep things in sync~+ not possible
- [x] check start commands, to make sure the configured ports are used, when starting the daemons
  - [x] adjust SCP
    - [x] make sure the buttons are not permanently redirecting to web interface on port 80, but use the dynamically configured ports (#363)
  - [ ] adjust webinterface
  - [ ] adjust batch
- [x] add buttons to reset ports to default numbers
##

**Comment by klonos** [comment]
You might want to add another task to the list: add buttons to reset ports to default numbers.

**Comment by jakoch** [comment]
thanks - added!

**Comment by jakoch** [comment]
Blocked by #623

**Comment by jakoch** [comment]
- mongo settings are read and written from mongod.conf now
- edited installers files to include the yaml-based "mongod.conf"
- the both paths (log path and dbpath) use "C:\server" as a placeholder
- used ReplaceStringInFile to replace C:\server with app path during installation
